content/developers/index.md

```diff
@@ -1,7 +1,148 @@
 ---
 title: Quick start
-description: Our mission as Mysterium Node runners
+description: How to setup your environment
 ---
 
 ## Quick start for developers
-...
+
+Setting up your environment for building, running and testing Mysterium node doesnt have to be hard.
+The minimum required software to set up is:
+
+| Software           | Minimum Version |
+|--------------------|-----------------|
+| Go                 | 1.14.0+         |
+| Docker             | 19.0+           |
```

Comment:
If you have more specific versions for these, let me know

Reply:
You could reword:  
`The minimum required software to set up for running, building and testing Mysterium node is:`  
to
`The minimum recommended pre-requisites for running, building and testing Mysterium node are:`  

This should imply that other version work but it's just not recommended